packages/orbit-components/src/Hide/Hide.stories.tsx

```diff
@@ -1,70 +1,65 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
-import Separator from "../Separator";
-import ChevronBackward from "../icons/ChevronBackward";
 import type { Devices } from "../utils/mediaQuery";
 
 import Hide from ".";
 
-export default {
-  title: "Hide",
-};
-
-export const WithSeparator = ({ block }) => {
-  return (
-    <Hide on={["largeMobile"]} block={block}>
-      <Separator />
-    </Hide>
-  );
+type HidePropsAndCustomArgs = React.ComponentProps<typeof Hide> & {
+  largeDesktop: boolean;
+  desktop: boolean;
+  tablet: boolean;
+  largeMobile: boolean;
+  mediumMobile: boolean;
+  smallMobile: boolean;
 };
 
-WithSeparator.story = {
+const meta: Meta<HidePropsAndCustomArgs> = {
+  title: "Hide",
+  component: Hide,
   parameters: {
-    info: "Configuration with Separator, for separator to work correctly block property has to be set.",
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["on"],
+    },
   },
-};
 
-WithSeparator.args = {
-  block: true,
+  args: {
+    largeDesktop: false,
+    desktop: false,
+    tablet: false,
+    largeMobile: false,
+    mediumMobile: false,
+    smallMobile: false,
+    block: false,
+  },
 };
 
-export const Playground = ({
-  largeDesktop,
-  desktop,
-  tablet,
-  largeMobile,
-  mediumMobile,
-  smallMobile,
-  block,
-}) => {
-  const on = [
-    largeDesktop && "largeDesktop",
-    desktop && "desktop",
-    tablet && "tablet",
-    largeMobile && "largeMobile",
-    mediumMobile && "mediumMobile",
-    smallMobile && "smallMobile",
-  ].filter(item => typeof item === "string") as Devices[];
+export default meta;
 
-  return (
-    <Hide on={on} block={block}>
-      <ChevronBackward />
-    </Hide>
-  );
-};
+type Story = StoryObj<HidePropsAndCustomArgs>;
+
+export const Playground: Story = {
+  render: ({ block, ...args }) => {
+    const { largeDesktop, desktop, tablet, largeMobile, mediumMobile, smallMobile } = args;
+
+    const on = [
+      largeDesktop && "largeDesktop",
+      desktop && "desktop",
+      tablet && "tablet",
+      largeMobile && "largeMobile",
+      mediumMobile && "mediumMobile",
+      smallMobile && "smallMobile",
+    ].filter(item => typeof item === "string") as Devices[];
+
+    return (
+      <Hide block={block} on={on}>
+        <div style={{ height: 20, width: 300, background: "darkViolet" }} />
```

Comment:
Could we use a percentage width here? It doesn't look very nice on `smallMobile`

![image](https://github.com/user-attachments/assets/e8f41c05-020a-435c-8321-150be97387f8)

Reply:
I wanted to set width in `%`, but the parent wrapper component (`<Hide>`) doesn't have set width, so percentage value for width is not working when `display: inline-block` is set for Hide component.

<img width="1199" alt="image" src="https://github.com/user-attachments/assets/532fc2a8-2abf-49e4-b3f0-ecf58ec13f9b">

I can set `width: 100%` and `min-width: 100px` at least to have "something" for "inline-block" display property. 🤔